docs/development/testing.md

```diff
@@ -154,6 +154,23 @@ an `HTTP 400 error`.
 In that case, configure an alternate load method by referencing the name of the test BPMN diagram in the load configuration
 settings.
 
+### Performance tests
+For now these tests are defined under `/test/e2e/performance/` as the performance is being measured for complex(e2e) tasks:
+- display diagram
+- zoom in / zoom out loaded diagram
+
+#### Why?
+To have visibility if new changes have impact on current performance
+
+#### Results
```

Comment:
I would have put the **Results** section after the **Run performance tests** section instead.